crates/daphne-worker/src/aggregator/roles/helper.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use crate::aggregator::App;
+use daphne::roles::DapHelper;
+
+impl DapHelper for App {}
```

Comment:
Huh, I'm just now noticing that `DapHelper` is basically just a marker trait. Is there any value to keeping it around?

Reply:
I think there is, even if just to show the intent of an API, that it's supposed to be used by the helper only